src/components/DocumentationButton/index.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import {
+  render,
+  screen,
+  fireEvent,
+  cleanup
+} from '@testing-library/react';
+
+import DocumentationButton from '.';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn(() => ({
+    t: (key: string) => key
+  }))
+}));
```

Comment:
Is this mock needed at all? I would have assumed that it is handled by the [global mock](https://github.com/terrestris/shogun-gis-client/blob/main/jest.setup.js#L13-L22).